### PR TITLE
[Merged by Bors] - doc(README): add build instructions

### PR DESCRIPTION
**Pull request by fpvandoorn**
Mostly retrieved from https://leanprover.zulipchat.com/#narrow/stream/270676-lean4/topic/Self-contained.20native.20toolchain

### CODE REVIEW

**Comment by digama0** [comment]
Why `make` and not `lake`? I've been using `lake [build]` and it seems to work.

**Comment by fpvandoorn** [comment]
For me `lake` without arguments does nothing. `lake build` does the same as `make build` AFAICT.

**Comment by digama0** [comment]
I think we should recommend `lake` instead of `make` if possible. One fewer moving part is good.

File: README.md [hunk 1]
```diff
@@ -15,3 +15,29 @@ during this experimentation phase.
...
+  If this also fails, follow the instructions under `Regular install` [here](https://leanprover-community.github.io/get_started.html).
+* Install `lake`
+  ```
+  cd .. # go to a folder where you want to clone repositories
+  git clone https://github.com/leanprover/lake.git
+  cd lake
+  lake +Kha/lean4:v4.0.0-bundle-rc build
+  ./build/bin/lake # this should output some usage information
+  cd ../mathlib4
+  ```
```
Comment (gebner) [inline README.md:38]:
```suggestion
```

You should never install lake manually.  Lake is bundled with lean, and the bundled lake version is the only supported one to use.

Comment (fpvandoorn) [reply README.md]: Ah, thanks. I didn't realize that the lake I used was bundled with Lean. My mistake!

**Comment by fpvandoorn** [comment]
I replaced `lake build` by `make build`. What is the `lake` command corresponding to `make`? Or shall I keep that as `make`?

**Comment by gebner** [comment]
> I replaced lake build by make build. What is the lake command corresponding to make? Or shall I keep that as make?

The makefile will be removed eventually when someone rewrites the `test` target in lake.  For now `make` is fine.

**Pull request closed without merging**